Source/Blazorise/wwwroot/datePicker.js

```diff
@@ -42,6 +42,11 @@ export function initialize(dotnetAdapter, element, elementId, options) {
     const mutationObserver = new MutationObserver(mutationObserverCallback);
     mutationObserver.observe(document.getElementById(elementId), { attributes: true });
 
+    const disableDatesOptions = options.disabledDates || [];
+    const disableDaysOptions = options.disabledDays ? [function (date) {
```

Comment:
Just to make sure, does this trully take in an array of functions?

Reply:
Think it's safe to assume that the date they pass in is valid? If we usually null check everything that's ours, then something from a 3rd party lib which we might not be sure, it's probably safer if we do instead of trusting it?

Reply:
Yes, the lib work with date values and with functions. Any they can be mixed.